Tagging.py

```diff
@@ -101,9 +101,20 @@
 
 join.columns = ['trans', 'tag1', 'tagml']
 
+for i in range (0, len (join.index)):
```

Comment:
there's a python style guide called PEP8 that defines industry-level formatting that people abide by.

one of those is to have no space after function names. in this case, it should be `len(join.index)` instead of `len (join.index)`

also see lines 66 and 94 above